docs/UserGuide.md

```diff
@@ -118,6 +118,54 @@ Showed all workouts in list
 ```
 ---
 
+### Update a Workout: `workout /update`
+Modifies an existing workout from the workout list.<br>
+*Noted: Only the number of repetition can be changed.*
+
+Format: `workout /update <index of workout to update> <new number of repetitions>`
+
+| Parameters                     | Description                                              |
+|--------------------------------|----------------------------------------------------------|
+| `<index of workout to update>` | The index of workout to be update from the workout list. |
+| `<new number of repetitions>`  | The number of repetitions to be changed to.              |
+
+**Example**<br>
+Initially, the workout list contains the following workouts:
+```
+----------------------------------------------------------------------
+Showing workouts 1-3 of 3:
+1. push up (10 reps)
+2. sit up (10 reps)
+3. pull up (10 reps)
+Showed all workouts in list
+----------------------------------------------------------------------
+```
+To update workout with index 1 to 15 reps, the following command should be entered:
```

Comment:
Perhaps the second part of the sentence could be rephrased to '..., enter the following command:'? The original phrase sounds like a suggestion, which sounds a bit off in this context.

Reply:
Thank you for your suggestions.